opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/StixCoreRelationshipCreation.tsx

```diff
@@ -643,62 +694,36 @@ class StixCoreRelationshipCreation extends Component {
         </span>
       </div>
     );
-  }
+  };
 
-  render() {
-    const { open, fromObject, toObjects, classes, t } = this.props;
-    const { step, displayProgress, progress } = this.state;
-    return (
-      <>
-        <Drawer
-          open={open}
-          anchor="right"
-          elevation={1}
-          sx={{ zIndex: 1202 }}
-          classes={{ paper: classes.drawerPaper }}
-          onClose={this.handleClose.bind(this)}
-        >
-          {step === 0
+  return (
+    <>
+      <Drawer
```

Comment:
For lighter DOM you could unmount Drawer if not open
```suggestion
      {open && (<Drawer
```